Fibonache.js

```diff
@@ -0,0 +1,32 @@
+// with Recursion. it works once, need to reset resultFibonachhiRecursion each time 
+function fibonachhiRecursion(countOfNumbers) {
+  if (countOfNumbers < 2) throw new Error("countOfNumbers should be at least 2");
```

Comment:
We know 0th and 1st fibonacci numbers so it's okay to have fibonacci numbers' array length to be equal to 1 or 2